File: Library.UI/src/app/models/book.ts
export interface Book {
    id: number;
    author: string;
    title: string;
    quantity: string;
    availableCount?: number;
}

export interface Reader {
    id: number;
    firstName: string;
    lastName: string;
    phone: string;
}

export interface Card {
    id: number;
    readerId: number;
    bookId: number;
    author: string;
    title: string;
    firstName: string;
    lastName: string;
    phone: string;
    givenDate: Date;
    returnDate: Date
}

File: Library.BL/Models/Book.cs
﻿using System;

namespace Library.BL.Models
{
    public class Book
    {
        public Guid Id { get; set; }

        public string Author { get; set; }

        public string Title { get; set; }

        public int Quantity { get; set; }

        public int AvailableCount { get; set; }
    }
}